packages/root-config/src/ontotext-root-config.js

```diff
@@ -65,6 +67,18 @@ layoutEngine.activate();
 
 defineCustomElements();
 
+const loadLicense = () => {
+  const licenseContext = ServiceProvider.get(LicenseContextService);
+  ServiceProvider.get(LicenseService).getLicense().then(license => {
+    licenseContext.updateLicense(license);
+  }).catch(() => {
+    licenseContext.updateLicense({
```

Comment:
Better create a local variable for this with the Partial<License> type or maybe cast it directly.

Reply:
Ignored as discussed, since this is JS